src/Exception/InsufficientPermissionsException.php

```diff
@@ -0,0 +1,6 @@
+<?php
+namespace Bernard\Exception;
```

Comment:
New line before namespace. Also, I would probably move this class to the FlatFile namespace